File: C++/2311.cpp
#include <iostream>
using namespace std;

int main(){
	int quantidade=0;
	float grau = 0.0, notas[7], soma=0.0, maior=0.0, menor=10.0;
	string nome;
	cin >> quantidade;
	while(quantidade > 0){
		cin >> nome >> grau;
		soma=0.0, maior=0.0, menor=10.0;
		for(int i=0;i<7;i++){
			notas[i]=0.0;
			cin >> notas[i];
			if(maior < notas[i]){
				maior = notas[i];
			}if(menor > notas[i]){
				menor = notas[i];
			}
		}
		int cont1=0, cont2=0;
		for(int j=0;j<7;j++){
			if(notas[j] == maior and cont1 == 0){
				cont1=1;
			}else if(notas[j] == menor and cont2 == 0){
				cont2=1;
			}else{
				soma = soma + notas[j];
			}	
		}
		soma = soma*grau;
		cout.precision(2);
		cout << nome << " " << fixed << soma << endl;
		quantidade--;
	}
	return 0;
}

File: Python/2311.py
quantidade, menor, maior, soma = 0,0,0,0
quantidade = int(input())
while(quantidade>0):
	menor, maior, soma, final= 10,0,0,0
	nome = input()
	dificuldade = float(input())
	notas = input().split()
	notas = list(notas)
	for i in range(len(notas)):
		notas[i] = float(notas[i])
	notas.sort()
	cont = 0
	for i in range(len(notas)-1):
		if(cont != 0):
			soma +=notas[i]
		else:
			cont+=1
	final = soma*dificuldade
	print(nome + " {:.2f}".format(final))
	quantidade-=1